### DISCUSSION TITLE
Restrict access to folders by default

### DISCUSSION THREAD

**Discussion by Narek-Daghlaryan** [drakkan/sftpgo#697] [Q&A] [answered]
Hello, 

Is there a way to restrict access to folders in an Azure Blob Storage scenario by default ? I know I can change permissions of the folder separately, but in my scenario I have a lot of folders and only some of them should be accessible. Also, is there a possibility to hide specific folders, so they are not visible when accessing through FileZilla.

Thank you!

**Comment by drakkan** [comment, marked as answer]
Hi,

permissions are inherited, so you can set the wanted permissions on parent folders and eventually override them for some child folders.
If you try the development version there is also an option to hide file/folders but please note that it may cause performance issues for large directories

**Reply by Narek-Daghlaryan** [reply]
Thank you very much! That solved our problem.
